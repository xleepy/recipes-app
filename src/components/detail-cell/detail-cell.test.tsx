import { render } from '@testing-library/preact';
import { describe, expect, it } from 'vitest';
import { DetailCell } from '.';

describe('Detail cell tests', () => {
  it('should render detail cell', () => {
    const { asFragment } = render(<DetailCell>test</DetailCell>);
    expect(asFragment()).toMatchSnapshot();
  });
});
