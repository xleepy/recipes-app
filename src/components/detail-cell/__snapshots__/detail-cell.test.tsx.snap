// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Detail cell tests > should render detail cell 1`] = `
<DocumentFragment>
  <div
    class="flex-1 _detail-cell_eb6d6d  "
  >
    test
  </div>
</DocumentFragment>
`;
